docs/Platform-Architecture/platform-azure-services/ai-services.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Azure AI Services
+summary: Design for AI services in Azure that are used offered for the platform.
+uri: https://defra.github.io/adp-documentation/Platform-Architecture/platform-azure-services/ai-services/
+authors:
+    - Logan Talbot
+date: 2024-05-15
+weight: 1
+---
+
+!!! note "TODO"
+    This page is a work in progress and will be updated in due course.
+
+
+# Azure AI Services
+
+This article details the AI Services Architecture for the solution at a high level.
+
+AI Services supported by ADP:
+
+- [Azure Open AI Service](#azure-open-ai)
+- [Azure AI Search](#azure-ai-search)
+
+!!! warning
+
+    Please ensure you fellow DEFRA's guidelines and policies when using AI services. This includes the use of data and the use of AI services in general in order to ensure your delivery project is using AI responsibly.
+
+## Azure Open AI
+![Azure Open AI](../../images/logos/openai.png){ align=left width="150" }Azure OpenAI Service provides REST API access to OpenAI's powerful language models including the GPT-4, GPT-4 Turbo with Vision, GPT-3.5-Turbo, and Embeddings model series. In addition, the new GPT-4 and GPT-3.5-Turbo model series have now reached general availability. These models can be easily adapted to your specific task including but not limited to content generation, summarization, image understanding, semantic search, and natural language to code translation.
```

Comment:
@LoganDavidTalbot  not sure why, { align=left width="150" } is displaying in the rich diff version. maybe the syntax is not recognised in markdown?